kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ViewRegistry.kt

```diff
@@ -80,10 +94,15 @@ class ViewRegistry private constructor(
     container: ViewGroup? = null
   ): View {
     @Suppress("UNCHECKED_CAST")
-    return (bindings[initialRendering::class] as? ViewBinding<RenderingT>)
-        ?.buildView(this, initialRendering, contextForNewView, container)
+    return (allBindings[initialRendering::class] as? ViewBinding<RenderingT>)
+        ?.buildView(this, initialRendering, contextForNewView, container)?.apply {
```

Comment:
Nit: This is a bit hard to read, consistent line breaks (before `?.`) might help?

Reply:
Much better.